packages/pycdfpp/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: pycdfpp
+  version: 0.8.3
+  top-level:
+    - pycdfpp
+source:
+  url: https://github.com/SciQLop/CDFpp/archive/refs/tags/v0.8.3.tar.gz
```

Comment:
We prefer downloading the wheel from PyPI rather than from GitHub commit. It seems like you don't upload Package sdist to PyPI, is there some reason behind it?